Libs/MRML/Core/vtkMRMLBitStreamVolumeNode.cxx

```diff
@@ -0,0 +1,235 @@
+#include "vtkMRMLBitStreamVolumeNode.h"
+
+// OpenIGTLink includes
+//#include "igtlioVideoDevice.h"
+
+// MRML includes
+#include "vtkMRMLScene.h"
+#include "vtkMRMLNRRDStorageNode.h"
+
+// VTK includes
+#include <vtkCollection.h>
+#include <vtkMatrix4x4.h>
+#include <vtkNew.h>
+#include <vtkObjectFactory.h>
+#include <vtkXMLUtilities.h>
+
+//----------------------------------------------------------------------------
+vtkMRMLNodeNewMacro(vtkMRMLBitStreamVolumeNode);
+
+//----------------------------------------------------------------------------
+// vtkMRMLBitStreamVolumeNode methods
+
+//-----------------------------------------------------------------------------
+vtkMRMLBitStreamVolumeNode::vtkMRMLBitStreamVolumeNode()
+{
+  this->compressionDevice = NULL;
```

Comment:
Per VTK convention, private member should start with upper case letter. ->`CompressionDevice`